pkg/lib/hooks.ts

```diff
@@ -19,7 +19,7 @@
 
 import cockpit from 'cockpit';
 import { useState, useEffect, useRef, useReducer } from 'react';
-import deep_equal from "deep-equal";
+import { dequal } from 'dequal';
```

Comment:
Can we use /lite here as well? IMHO it'd be better to just have *one* function in the bundle, not two. We don't need the more "exotic" data types that full dequal supports.

Reply:
Well it messed up typescript and I am kinda done with dealing with typescript but yes *ugh* blocker it is then.

Reply:
By some miracle, typescript is now happy, so I am happy.

Reply:
Sorry for the confusion -- I didn't mean "typescript" here. This is just a decision whether or not you need the additional features of "dequal" for comparing e.g. `ArrayBuffer` or `DataView` objects (that's what I meant with "exotic data types")